src/render.tsx

```diff
@@ -100,10 +101,20 @@ function buildRenderResult(
     ...getQueriesForElement(instance),
     update,
     unmount,
-    container: instance,
     rerender: update, // alias for `update`
     toJSON: renderer.toJSON,
     debug: debug(instance, renderer),
+    get root() {
+      return getHostChildren(instance)[0];
+    },
+    UNSAFE_root: instance,
+    get container() {
+      if (!getConfig().useBreakingChanges) {
+        return instance;
```

Comment:
should we log a warning to tell the user `container` will be deprecated in the futur in favor of root?